testing/profiler-tests/build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+  id 'java'
+}
+
+dependencies {
+  testImplementation("org.testcontainers:testcontainers:1.15.3")
+}
+
+tasks.register('copyProfilingAgentJar', Copy) {
+  from "../../agent/build/libs/"
+  include "splunk-otel-javaagent-*-all.jar"
+  into "build"
+}
+
+copyProfilingAgentJar.dependsOn(':agent:assemble')
+test.dependsOn(copyProfilingAgentJar)
```

Comment:
WDYT about using the smoke-tests way of getting the agent jar?

```groovy
tasks.test {
  def shadowTask = project(":agent").tasks.named("shadowJar").get()
  inputs.files(layout.files(shadowTask))

  doFirst {
    jvmArgs("-Dio.opentelemetry.smoketest.agent.shadowJar.path=${shadowTask.archiveFile.get()}")
  }
}
```

Reply:
That's cool, I can use that for consistency.  How do you ensure that the agent jar is built first tho?

Reply:
nm, separate concern.  I love it, ended up working out great!